app/views/content/is-teaching-right-for-me/qualifications-you-need-to-teach.md

```diff
@@ -54,12 +54,15 @@ This could be through having a degree in the subject, or having:
 * a degree related to but not in the subject – for example, your degree is in engineering but you’d like to teach physics
 * an unrelated degree but relevant professional experience
 
+You can also train to teach two subjects at primary or secondary. You can:
+
+* choose primary with a subject, this will allow you to teach more of that other subject in your primary school
+* choose two subjects at secondary, this will allow you to teach both subjects in your secondary school
```

Comment:
```suggestion
* choose two subjects at secondary, which will allow you to teach both subjects in your secondary school
```
@MaxineCl again, not sure about `allow`

Reply:
@gemmadallmandfe for some reason I can't add the comment by the line, the + won't open but I don't think the wording of this sentence is quite right? **You can also train to teach two subjects at primary** or secondary. 

Looking on Find you can apply to train in primary with a specialist subject but this says train to teach 2 subjects and primary isn't a subject?